addons/website_event_track/views/event_track_templates_proposal.xml

```diff
@@ -33,79 +33,83 @@
                             </p>
                         </section>
                     </div>
-                    <div t-if="not track">
-                        <section class="mt-5">
-                            <h3 class="o_page_header">Application</h3>
-                            <p>
-                                Fill this form to propose your talk.
-                            </p>
-                            <div class="alert alert-info" role="alert">
-                                <i class="fa fa-info-circle"/>
-                                You can add multiple speakers by separating names, emails and phones with commas.
-                            </div>
-                        </section>
-                        <section id="forms" class="mb-5"
+                    <div class="d-flex justify-content-center" t-if="not track">
+                        <section id="forms" class="col-lg-9 col-md-10 mb-5"
                             t-if="event.website_track_proposal">
-                            <form class="mt32 js_website_submit_form" t-attf-action="/event/#{event.id}/track_proposal/post" method="post" enctype="multipart/form-data">
+                            <form class="mt32 js_website_submit_form o_website_event_track_proposal_form" t-attf-action="/event/#{event.id}/track_proposal/post" method="post" enctype="multipart/form-data">
                                 <input type="hidden" name="csrf_token" t-att-value="request.csrf_token()"/>
-                                <div class="form-group row">
-                                    <label class="col-lg-3 col-md-4 col-form-label" for="track_name">Talk Title</label>
-                                    <div class="col-lg-7 col-md-8">
-                                        <input type="text" class="form-control" name="track_name" required="True"/>
+                                <div id="track_intro">
+                                    <h4 class="mt32">
+                                    <b>Talk Intro</b>
```

Comment:
Some indentation issues here (you may re-check the whole diff : you can also concatenate some lines to lessen diff like 
```
<h4 class="mt32"><b>Talk Intro</b></h4>
```